src/Features/Core/Portable/GenerateConstructorFromMembers/AbstractGenerateConstructorFromMembersCodeRefactoringProvider.GenerateConstructorWithDialogCodeAction.cs

```diff
@@ -76,10 +76,12 @@ public override object GetOptions(CancellationToken cancellationToken)
                 var addNullChecksOption = result.Options.FirstOrDefault(o => o.Id == AddNullChecksId);
                 if (addNullChecksOption != null)
                 {
+                    // ILegacyGlobalOptionsWorkspaceService is guaranteed to be not null here because we have checked it before the code action is provided.
+                    var globalOptions = _document.Project.Solution.Services.GetRequiredService<ILegacyGlobalOptionsWorkspaceService>();
```

Comment:
Ideally we should also check the IPickMemberService. but right now we filter out CodeActionWithOptions
https://github.com/dotnet/roslyn/blob/e3f89a400c23c0a68e4f7882af2e3692673aff2e/src/Features/LanguageServer/Protocol/Handler/CodeActions/CodeActionHelpers.cs#L95
so it's not a problem now.
And to check PickMemberService it needs to refactor a lot of tests, so would like to do it later